tests/test_version_utils.py

```diff
@@ -0,0 +1,293 @@
+"""Tests for cvejob.version_utils module."""
+
+import pytest
+from cvejob.version_utils import (
+    get_configuration_nodes, VersionOperator, VersionSpec, VersionRange,
+    classify_versions, get_affected_ranges, get_safe_ranges, ClassifiedVersion,
+    get_upstream_versions, get_ranges_from_cve, get_version_ranges
+)
+
+
+def test_get_ranges_from_cve(maven_cve):
+    """Test get_ranges_from_cve()."""
+    affected, safe = get_ranges_from_cve(maven_cve, 'org.apache.tomcat:tomcat-catalina', 'java')
+    assert len(affected) == 3
+    affected_strings = [str(x) for x in affected]
+    assert len(safe) == 4
+    safe_strings = [str(x) for x in safe]
+
+    expected_affected = ('<=7.0.90,7.0.23', '<=8.5.33,8.5.0', '<=9.0.11,9.0.0.M1')
+    for item in expected_affected:
+        assert item in affected_strings
+        affected_strings.pop(affected_strings.index(item))
+
+    expected_safe = ('>=9.0.12', '<=8.5.38,8.5.34', '<=8.0.53,7.0.91', '<=7.0.22,7.0.0')
+    for item in expected_safe:
+        assert item in safe_strings
+        safe_strings.pop(safe_strings.index(item))
+
```

Comment:
what about tests for `get_ranges_from_cve(maven_cve, None,  'java')`
`get_ranges_from_cve(maven_cve, 'foobar',  None)` etc.

Just nitpicking, I'll take care of it ;)